examples/ppo/models.py

```diff
@@ -0,0 +1,51 @@
+import flax
+from flax import nn
+import jax.numpy as jnp
+
+class ActorCritic(flax.nn.Module):
+  '''
+  Architecture from "Human-level control through deep reinforcement learning."
+  Nature 518, no. 7540 (2015): 529-533.
+  Note that this is different than the one from  "Playing atari with deep
+  reinforcement learning." arxiv.org/abs/1312.5602 (2013)
+  '''
+  def apply(self, x, num_outputs):
+    x = x.astype(jnp.float32) / 255.
+    dtype = jnp.float32
+    x = nn.Conv(x, features=32, kernel_size=(8, 8),
+                strides=(4, 4), name='conv1',
+                dtype=dtype)
+    # x = nn.relu(x)
```

Comment:
On exactly one combination of jax, jaxlib and flax versions that I have once used on Colab, `relu` was hitting AssertionError in `jax/interpreters/partial_eval.py`, hence the `jnp.maximum`. I think now it can be switched back to `relu` safely, thanks for spotting this!

Reply:
Since https://github.com/google/flax/pull/470/commits/f398660136ffbf796f8a1a6e191605887b259429, `nn.relu` is not commented out and `jnp.maximum` is not used. 

For future reference: `nn.relu` was causing a bug for this combination of versions: Python 3.7.6, jax 0.1.75, jaxlib 0.1.51, and flax 0.2.0. Upgrading to flax 0.2.1. solved the problem.